.github/workflows/deploy-docs.yml

```diff
@@ -19,25 +15,17 @@ defaults:
     working-directory: docs
 
 jobs:
-  build:
-    if: github.event_name == 'pull_request'
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Build with mkdocs-material via Docker
-        run: docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build --strict
-
   deploy:
     if: github.event_name == 'push'
+    name: 'Deploy Docs'
```

Comment:
Another esthetic optinion: I would not repeat the word Docs here as the name of the whole file is already "Documentation". So it will look like: Documentation / Deploy Docs which is kinda repetetive. Maybe better choice is: Deocumentation / Publish to GitHub Pages (or only Publish).

The Build is fine imho with: Documentation / Verify Build

Reply:
That's a good point!

I'm curious if it's better to have a PR and Push workflows that are generic, such that it includes the doc relevant jobs, but could have others such as lint. Or if those are better off as separate that's fine too and I'll just adjust the name to "Publish to Github Pages".